Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Batch.java

```diff
@@ -102,7 +102,7 @@ public Batch(String title, Collection<? extends Process> processes) {
     }
 
     /**
-     * The function getTitle() returns the batch title. Using titles for batches
```

Comment:
Actually if we want to be so English correct - there is simple rule: _The definition of a sentence is a subject and verb._ So _The function getTitle() returns the batch title_ is correct and _Returns the batch title._ is grammatically incorrect. There is one exception when you can make sentence without subject - imperative, but imperative here would be _Return_, not _Returns_.